src/lib/bootloader/grub2base.rb

```diff
@@ -53,110 +61,108 @@ def pmbr_setup(action, *devices)
       end
     end
 
-    # Propose global options of bootloader
-    def StandardGlobals
-      # s390 do not have os_prober, see bnc#868909#c2
-      # ppc have slow os_prober, see boo#931653
-      disable_os_prober = (Arch.s390 || Arch.ppc) ||
-        ProductFeatures.GetBooleanFeature("globals", "disable_os_prober")
-      {
-        "timeout"   => "8",
-        "default"   => "0",
-        "vgamode"   => "",
-        "gfxmode"   => "auto",
-        "terminal"  => Arch.s390 ? "console" : "gfxterm",
-        "os_prober" => disable_os_prober ? "false" : "true",
-        "activate"  => Arch.ppc ? "true" : "false"
-      }
+    def read(reread: false)
+      grub_default.load if !grub_default.loaded? || reread
+      grub_cfg = CFA::Grub2::GrubConf.new
+      grub_cfg.load
+      @sections = ::Bootloader::Sections.new(grub_cfg)
     end
 
-    # Update read settings to new version of configuration files
-    def Update
-      Read(true, true)
+    def write
+      super
+      grub_default.save
+      # TODO: call grub_install
```

Comment:
because perl-Bootloader call it. Goal of this branch is to remove pbl dependency.